code/__DEFINES/is_helpers.dm

```diff
@@ -37,7 +37,13 @@
 
 #define isspaceturf(A) istype(A, /turf/space)
 
-#define isobserver(A) istype(A, /mob/abstract/observer)
+#define isghost(A) istype(A, /mob/abstract/ghost)
+
+#define isobserver(A) istype(A, /mob/abstract/ghost/observer)
+
+#define isstoryteller(A) istype(A, /mob/abstract/ghost/storyteller)
+
+#define isabstract(A) istype(A, /mob/abstract)
```

Comment:
Given we use abstract as a metaproperty, the name of this define is confusing and should probably be something like ismobabstract or similar thing